src/Evolution/Systems/GeneralizedHarmonic/GaugeSourceFunctions/InitializeDampedHarmonic.hpp

```diff
@@ -60,6 +61,28 @@ struct DampedHarmonicSpatialDecayWidth {
       "gauge."};
   using group = gauges::OptionTags::GaugeGroup;
 };
+
+/// The amplitudes for the L1, L2, and S terms, respectively, for the damped
+/// harmonic gauge.
+template <typename Frame>
+struct DampedHarmonicAmplitudes {
```

Comment:
(just picking a spot that's reasonable)

I generally dislike the whole way the gauge parameters are handled as a bunch of separate things. This is for a couple reasons:
- It makes them seems like distinct entities on their own, which they are
- It means we have about 5 or so options tags and compute tags instead of 1

I would like to see all the options be in a single struct:
```cpp
template <bool UseRollonGauge>
struct DhGaugeParameters {
  int exponent_1;
  int exponent_2;
  ...
  using options = tmpl::list<...>;
};
```
Then we have a single object to pass around that contains all the parameters, and is constructed from options.

The whole idea of the option groups was not really for this purpose, but to group related options consumed by different classes in the input file. Here we have a bunch of options consumed by the exact same code and so that should just be a single struct

Reply:
Sorry it took me so long to get back to this, @nilsdeppe ! I had some fires to put out this week, but I finally was able to push a fixup commit. Is the fixup on the right track for what you had in mind?